### ISSUE TITLE
Social sharing icons not visible on home page - Twenty Twenty-Two theme

### ISSUE THREAD

**Issue by pavanpatil1** [rtCamp/rtsocial#96]
```
Environment: 
WordPress: v5.9.1
BuddyPress: v10.1.0
rtMedia: v4.6.10
rtSocial: develop branch. 
```

On Twenty Twenty-Two theme the social sharing icon is not visible. 

**Expected Result:** Social sharing icons should be visible in front of posts. 

![image](https://user-images.githubusercontent.com/44057535/157593640-dbf4eed0-e9c7-4235-8688-23328e50573b.png)

Actual  result          |  Expected result
:----------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/44057535/157593640-dbf4eed0-e9c7-4235-8688-23328e50573b.png)| ![image](https://user-images.githubusercontent.com/44057535/157593897-60af96ea-5ac3-4802-b3ae-3f3abaa50405.png)

**Comment by pavanpatil1** [comment]
```
Environment:
WordPress: v5.9.1
rtSocial: PR #100 

```

Cross-checked the issue. The fix is working fine.

![image](https://user-images.githubusercontent.com/44057535/157801221-8e3b9513-b635-4454-9d68-377bb2c06ec1.png)

**Comment by pavanpatil1** [comment]
Fixed and released in v2.2.3
